src/containers/Header/Search.tsx

```diff
@@ -36,9 +38,24 @@ const determineHashType = async (id: string, rippledContext: XrplClient) => {
     await getTransaction(rippledContext, id)
     return 'transactions'
   } catch (e) {
+    const isAmendment = await isAmendmentType(id)
```

Comment:
Maybe this should happen in parallel since there are 3 options.  This is even further complicated by #544.  Doing in parallel makes sense to allow for #831. I think this should be moved out into a new PR so these other issues can maybe be addressed there too.